test/JDBC/upgrade/14_6/schedule

```diff
@@ -362,3 +362,4 @@ Test-sp_babelfish_volatility
 BABEL_SCHEMATA
 BABEL-3657
 BABEL-3938
+binary-index
```

Comment:
Please also put this in 14_7/schedule file.

Reply:
added